llm/python/README.md

```diff
@@ -0,0 +1,113 @@
+* `--output_dir` - output directory for saving OpenVINO model
+* `--precision` - (optional, default FP32), precision for model conversion FP32 or FP16
+* `--save_orig` - flag for saving original pytorch model, model will be located in `<output_dir>/pytorch` subdirectory.
+* `--compress_weights` - flag for saving model with compressed weights.
+* `--compress_weights_backends` - (optional, default openvino) backends for weights compression, this option has an effect only with `--compress_weights`. For `openvino` backend, model will be located in `<output_dir>/pytorch/dldt/INT8_compressed_weights` subdirectory. For `pytorch` backend, model will be located in `<output_dir>/pytorch/dldt/PT_compressed_weights` subdirectory. You can specify multiple backends separated by a space.
+
+Usage example:
+```bash
+python convert.py --model_id meta-llama/Llama-2-7b-chat-hf --output_dir models/llama-2-7b-chat
+```
+
+the result of running command will have following file structure:
+
+    |-llama-2-7b-chat
+      |-pytorch
+        |-dldt
+           |-FP32
+              |-openvino_model.xml
+              |-openvino_model.bin
+              |-config.json
+              |-added_tokens.json
+              |-tokenizer_config.json
+              |-tokenizer.json
+              |-tokenizer.model
+              |-special_tokens_map.json
+
+(3) Bechmarking
+Prerequisites:
+install benchmarking dependencies using requirements.txt
+
+``` bash
+pip install -r requirements.txt
+```
+notes: **You can specify the installed openvino version through pip install**
+``` bash
+# e.g. 
+pip install openvino==2023.0.0 or pip install openvino-dev==2023.0.0
+```
+
+(4) Run the following command to test the performance of one LLM model
+``` bash
+python benchmark.py -m <model> -d <device> -r <report_csv> -f <framework> -p <prompt text> -n <num_iters>
+```x
+# e.g.
+python benchmark.py -m models/llama-2-7b-chat/pytorch/dldt/FP32 -n 2
+python benchmark.py -m models/llama-2-7b-chat/pytorch/dldt/FP32 -p "What is openvino?" -n 2
+python benchmark.py -m models/llama-2-7b-chat/pytorch/dldt/FP32 -pf prompts/llama-2-7b-chat_l.jsonl -n 2
+```
+Parameters:
+* `-m` - model path
+* `-d` - inference device (default=cpu)
+* `-r` - report csv
+* `-f` - framework (default=ov)
+* `-p` - interactive prompt text
+* `-pf` - path of JSONL file including interactive prompts
+* `-n` - number of benchmarking iterations, if the value greater 0, will exclude the first iteration. (default=0)
+
+``` bash
+python ./benchmark.py -h # for more information
+```
+
+## Running `torch.compile()`
+
+The option `--torch_compile_backend` uses `torch.compile()` to speed up the PyTorch code by compiling it into optimized kernels using a selected backend.
+
+Prerequisites: install benchmarking dependencies using requirements.txt
+
+``` bash
+pip install -r requirements/requirements.txt
+```
+
+In order to run the `torch.compile()` on GPU, install additionally the nightly PyTorch version:
```

Comment:
need some wording clarification, because it is probably about running torch.compile for torch with cuda gpu?
There is no need in cuda kernels if benchmark runs with openvino GPU.

Reply:
updated